scriptworker/version.py

```diff
@@ -52,7 +52,7 @@ def get_version_string(version):
 
 # 1}}}
 # Semantic versioning 2.0.0  http://semver.org/
-__version__ = (17, 2, 2)
+__version__ = (17, 2, 3)
```

Comment:
We're changing the arguments to functions in a non-backwards compatible way, so this would be `18.0.0`.